presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -102,8 +102,67 @@ Property Name                                      Description
 ``hive.immutable-partitions``                      Can new data be inserted into existing partitions?           ``false``
 
 ``hive.max-partitions-per-writers``                Maximum number of partitions per writer.                     100
+================================================== ============================================================ ==========
+
+Amazon S3 Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The Hive connector also allows querying data in Hive where data files are stored in Amazon S3. 
```

Comment:
I think just say "The hive connector allows querying data that is stored in Amazon S3"